cpp_config_template.yml

```diff
@@ -21,7 +21,7 @@ jobs:
                 docker:
                         - image: cnerg/ubuntu-18.04_pyne-3:latest #any public docker container
                 steps:
-                        -run: git push origin master
+                        -run: git push origin main
```

Comment:
I agree with this change, but I'm not sure I think it's wise to have a `pushing` job in this config file.  It seems either dangerous or unnecessary, or both - especially for the beginners for whom these templates are intended.

Perhaps @psbritt can remind us why it's here?